### ISSUE TITLE
Seq++ causes VCV to hang on startup after any patch adjustment + doesn't run

### ISSUE THREAD

**Issue by geslotencirkel** [squinkylabs/SquinkyVCV#149]
Seems Seq++ saved the midi sequence correctly, but the way it is patched is causing VCV to not play on patch load and any rack content changes (deletion, cable unplugging) will cause VCV to freeze and hang. I assume its because of the way it is cabled in the patch - Impromptu is always running and sending a clock to Seq++, there are also a lot of Stoermelder Strips that are scattered in the rack, if that matters.
Haven't tried reproducing a consistent crash case, will update if I find anything while trying to recover the patch.

**Comment by geslotencirkel** [comment]
ended up removing the squinkylabs folder, saving the vcv patch without it after loading and closing the alert that squinkylabs seq++ is missing, then installing squinky again and repatching the module and loading the midi file. Seems to work so far on patch open, so not sure what happened. Repatched it the same way as before. Will update if anything else comes up.

**Comment by squinkylabs** [comment]
Yes, please report if this happens again. It's not something that we've seen, so it may be elusive. thanks for reporting it.

**Comment by geslotencirkel** [comment]
my original file still acts the same. Probably too much work to load it and not aware of any debugger or crash log that could be generated.

**Comment by squinkylabs** [comment]
There is a log file generated by VCV that might have something interesting in it. on my windows machine it's in c:\Users\<my name>\Documents\Rack\log.txt.

btw: it's not normally a problem to have clocks running all time on patch load. And, while it certainly could be a Seq++ issue is there any reason not to suspect other plugins on this patch?

**Comment by geslotencirkel** [comment]
i've never had issue like this before - so always kept the Impromptu clock running on all patches when saving.
Not seeing anything useful in the log after clearing it and loading the bad file and tried recording a new sequence in another version of the file with a newly added seq++ readded - saved the file and was able to load it without issue.
will be vigilant. 
awesome module anyway.

**Comment by squinkylabs** [comment]
Glad that you like it. If you really get proficient with it you might try the "companion" 4X4 sequencer.
